tests/cameras/test_cameras.py

```diff
@@ -120,6 +120,18 @@ def test_pinhole_camera():
     pinhole_camera.generate_rays(camera_indices=0, coords=coords)
 
 
+def test_orthophoto_camera():
+    """Test that the orthophoto camera model works."""
+
+    c2w = torch.eye(4)[None, :3, :]
+    orthophoto_camera = Cameras(
+        cx=200.0, cy=400.0, fx=1.0, fy=1.0, camera_to_worlds=c2w, camera_type=CameraType.ORTHOPHOTO
+    )
+    camera_ray_bundle = orthophoto_camera.generate_rays(camera_indices=0)
+    assert isinstance(camera_ray_bundle, RayBundle)
+    assert camera_ray_bundle.shape == torch.Size([800, 400])
```

Comment:
Could you add to this test to guarantee that the ray origins are correct, especially after rotation?

Reply:
Oh, I discovered a bug related to rotation issues caused by incorrectly using a left-handed coordinate system. The special case I used during testing (rotating 90 degrees along the z-axis) led me to overlook this issue. Thank you for your review. I have fixed this problem in the latest commit.
![output](https://github.com/nerfstudio-project/nerfstudio/assets/99106973/a5095d58-7841-43b3-98ff-45b97d836879)
![rotated](https://github.com/nerfstudio-project/nerfstudio/assets/99106973/96bfa31e-3b33-42ec-a891-7d8cfc055d3a)

Reply:
Sounds great, I will add a more complete test case, and make it work for _generate_rays_from_coords.